Packs/CommonTypes/ReleaseNotes/3_2_22.md

```diff
@@ -1,4 +1,229 @@
 
+#### Indicator Fields
```

Comment:
we need only one release notes :)
I know that you need to update the RN every time, but lets have one file

Reply:
3_2_22 was a mistake, fixed it.